consensus/consensus_tests/utils_test.go

```diff
@@ -54,43 +57,37 @@ type IdToNodeMapping map[typesCons.NodeId]*shared.Node
 
 /*** Node Generation Helpers ***/
 
-var (
-	DefaultServiceURL    = "https://foo.bar" // TODO (team) cleanup consensus testing module with centralization of utilities
-	DefaultChainID       = "unittest_net"
-	DefaultStakeAmount   = types.BigIntToString(big.NewInt(1000000000))
-	DefaultAccountAmount = types.BigIntToString(big.NewInt(1000000000))
-)
-
-func GenerateNodeConfigs(_ *testing.T, numValidators int) (configs []*genesis.Config, genesisState *genesis.GenesisState) {
-	genesisState, keys := test_artifacts.NewGenesisState(numValidators, 1, 1, 1)
+func GenerateNodeConfigs(_ *testing.T, n int) (configs []modules.Config, genesisState modules.GenesisState) {
```

Comment:
Revert back to the two lines above.

1. Rename `n` to `numValidators`
2. genesisState and keys can be initialized on the same line.